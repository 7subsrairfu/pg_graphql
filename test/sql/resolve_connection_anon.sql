select jsonb_pretty(
    gql.dispatch($$
        {
          allAccounts {
            totalCount
            pageInfo{
                startCursor
                endCursor
                hasPreviousPage
                hasNextPage
            }
            edges {
              cursor
              node {
                id
                nodeId
                email
              }
            }
          }
        }
    $$)
);
