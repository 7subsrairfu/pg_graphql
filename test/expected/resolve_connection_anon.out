select jsonb_pretty(
    gql.dispatch($$
        {
          allAccounts {
            totalCount
            pageInfo{
                startCursor
                endCursor
                hasPreviousPage
                hasNextPage
            }
            edges {
              cursor
              node {
                id
                nodeId
                email
              }
            }
          }
        }
    $$)
);
                             jsonb_pretty                             
----------------------------------------------------------------------
 {                                                                   +
     "data": {                                                       +
         "allAccounts": {                                            +
             "edges": [                                              +
                 {                                                   +
                     "node": {                                       +
                         "id": 1,                                    +
                         "email": "aardvark@x.com",                  +
                         "nodeId": "WyJwdWJsaWMiLCAiYWNjb3VudCIsIDFd"+
                     },                                              +
                     "cursor": "WyJwdWJsaWMiLCAiYWNjb3VudCIsIDFd"    +
                 },                                                  +
                 {                                                   +
                     "node": {                                       +
                         "id": 2,                                    +
                         "email": "bat@x.com",                       +
                         "nodeId": "WyJwdWJsaWMiLCAiYWNjb3VudCIsIDJd"+
                     },                                              +
                     "cursor": "WyJwdWJsaWMiLCAiYWNjb3VudCIsIDJd"    +
                 },                                                  +
                 {                                                   +
                     "node": {                                       +
                         "id": 3,                                    +
                         "email": "cat@x.com",                       +
                         "nodeId": "WyJwdWJsaWMiLCAiYWNjb3VudCIsIDNd"+
                     },                                              +
                     "cursor": "WyJwdWJsaWMiLCAiYWNjb3VudCIsIDNd"    +
                 },                                                  +
                 {                                                   +
                     "node": {                                       +
                         "id": 4,                                    +
                         "email": "dog@x.com",                       +
                         "nodeId": "WyJwdWJsaWMiLCAiYWNjb3VudCIsIDRd"+
                     },                                              +
                     "cursor": "WyJwdWJsaWMiLCAiYWNjb3VudCIsIDRd"    +
                 },                                                  +
                 {                                                   +
                     "node": {                                       +
                         "id": 5,                                    +
                         "email": "elephant@x.com",                  +
                         "nodeId": "WyJwdWJsaWMiLCAiYWNjb3VudCIsIDVd"+
                     },                                              +
                     "cursor": "WyJwdWJsaWMiLCAiYWNjb3VudCIsIDVd"    +
                 }                                                   +
             ],                                                      +
             "pageInfo": {                                           +
                 "endCursor": "WyJwdWJsaWMiLCAiYWNjb3VudCIsIDVd",    +
                 "hasNextPage": false,                               +
                 "startCursor": "WyJwdWJsaWMiLCAiYWNjb3VudCIsIDFd",  +
                 "hasPreviousPage": false                            +
             },                                                      +
             "totalCount": 5                                         +
         }                                                           +
     },                                                              +
     "errors": [                                                     +
     ]                                                               +
 }
(1 row)

